src/Aquifer.Data/Entities/HelpDocumentEntity.cs

```diff
@@ -0,0 +1,22 @@
+namespace Aquifer.Data.Entities;
+
+public class HelpDocumentEntity
+{
+    public int Id { get; set; }
+    public required HelpDocumentType Type { get; set; }
+    public required string Title { get; set; }
+    public required string Url { get; set; }
```

Comment:
No need for `required` on entities.

Reply:
I would argue that `required` should be in place for the `Title` and `Url` as they are a must and we have no logic for setting a default value.

Reply:
Because it's the database's responsibility to enforce this. We don't know how we might want to use and build these entities. The default value for them should be `= null!;`.